tests/installation/addon_products_sle.pm

```diff
@@ -134,22 +134,6 @@ sub handle_addon {
     }
 }
 
-sub test_addonurl {
```

Comment:
I would suggest to adjust this check. It's really useful hint to check if the url is valid or not.

Reply:
i was thinking the same but as the URLs comes from the hardcoding variables(ADDONURL_*), i thought that it is not make sense to keep it. 

Lets think about it tho